src/views/auth/SignUp.vue

```diff
@@ -27,19 +27,12 @@
             type="password"
           ></el-input>
         </el-form-item>
-        <!-- <el-form-item prop="confirm">
-          <el-input
-            v-model="form.confirm"
-            placeholder="Confirm"
-            type="password"
-          ></el-input>
-        </el-form-item> -->
         <el-form-item>
           <el-button
             class="signup__button"
             type="success"
             block
-            @click="signUp('form')"
+            @click="onSubmit('form')"

```

Comment:
Same here... you need to simply use: this.$refs.form and that's all inside `onSubmit` function